bin/s3-sync.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+if [[ -z "$1" ]]; then
+    echo "Bucket name required"
+    exit 1
+fi
+
+SHORT_CACHE="--cache-control max-age=60" # 1 minute
```

Comment:
I used the 1 minute value suggested by @jwhitlock in #335 but I would prefer a longer time, say 5-15 minutes.